app/representers/api/episode_representer.rb

```diff
@@ -44,10 +44,16 @@ class Api::EpisodeRepresenter < Api::BaseRepresenter
   property :segment_count
   property :media_version_id, as: :media_version, writeable: false
 
+  property :medium
+
   collection :media,
     decorator: Api::MediaResourceRepresenter,
     class: MediaResource
 
+  property :uncut,
```

Comment:
Does this need some sort of custom setter (similar to `def media=`) to do the same logic?  Mainly:

- No-op if nil / not-sent
- Using the `MediaResource.build` helper to allow sending plain string `http://the/media/url` vs `{href: "..."}` vs `{originalUrl: "..."}`
- Only replacing if the original url has changed
- Somewhere to get/set the uncut segmentation array?